### PR TITLE
ci(make-release): optimize trigger

### PR DESCRIPTION
**Pull request by kerm1it**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [x] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English | - |
| 🇨🇳 Chinese | - |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

**Comment by kerm1it** [comment]
@xrkffgg release 的 ci 我加了触发条件，你看看对不对？不然每次一推分支都会发个运行失败的错误邮件。

**Review by xrkffgg — COMMENTED** [review]
好像不太对，我测试下

File: .github/workflows/make-release.yml [hunk 1]
```diff
@@ -8,6 +8,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: make release
+        if: github.event.create.ref_type == 'tag'
```
Comment (xrkffgg) [inline .github/workflows/make-release.yml:11]:
```suggestion
        if: github.event.ref_type == 'tag'
```

Comment (kerm1it) [reply .github/workflows/make-release.yml]: 这样好使么？我看文档里面是在 create 下面的？

**Comment by xrkffgg** [comment]
😂😂 你有收到 Actions 的邮件吗？我没收到。不过现在改了。Action 里我也改成提示了。

**Comment by kerm1it** [comment]
改了后，第一次推的时候就没有收到了

**Comment by kerm1it** [comment]
试试看看打 tag 的时候能不能正常触发

**Comment by xrkffgg** [comment]
试过了 https://github.com/xrkffgg/test-ci

**Pull request merged**